metrics_sdk/lib/opentelemetry/sdk/metrics/aggregation.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentelemetry/sdk/metrics/aggregation/histogram'
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      # The Aggregation module contains the OpenTelemetry metrics reference
+      # aggregation implementations.
+      module Aggregation
```

Comment:
Is this meant to be API-private, or does the spec want us to expose aggregations from the SDK?

Reply:
Looks like the answer is "expose". 👌